model/jpa/src/main/resources/META-INF/orm.xml

```diff
@@ -38,6 +38,9 @@ http://java.sun.com/xml/ns/persistence/orm_2_0.xsd"
         <discriminator-value>android</discriminator-value>
     </entity>
     <entity class="Installation" access="FIELD">
+        <table>
+            <index column-list="deviceToken"></index>
```

Comment:
I think this has impact on future migration scripts. I'd like to get the migration from @qmx in before we do merge this